reports/tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from reports.utils import haversine
+
+# from reports.utils import get_distance
+# @pytest.mark.parametrize(
+#     "lon1,lat1,lon2,lat2,distance_km",
+#     [
+#         [24.9384, 60.1699, 22.2666, 60.4518, 150.44]
+#     ],  # Helsinki  # Turku  # Distance in km
+# )
+# def test_get_distance(
+#     lon1: float, lat1: float, lon2: float, lat2: float, distance_km: float
+# ):
+#     assert get_distance(lon1, lat1, lon2, lat2) == distance_km
```

Comment:
I have no idea why this gives results something like 250km (Helsinki - Turku). Can the 3D calculator make so much difference? It seems like the Google Map is using 2D and the same haversine -algorithm (great circle). I think it's ok to use it and did not need any external library